### ISSUE TITLE
Definition of MeshMark

### ISSUE THREAD

**Issue by leecalcote** [service-mesh-performance/service-mesh-performance#227]
### Description: 

Create MeshMark provides a universal performance index to gauge your mesh’s efficiency against deployments in other organizations’ environments. MeshMark functions as a service mesh performance index (a scale) to provide people the ability to weigh the value of their service mesh versus the overhead of their service mesh and assess whether they are getting out of the mesh what they are “paying” for in it.

Work with maintainers from Layer5, Intel, Red Hat, and HashiCorp on researching cloud native infrastructure performance. Use machine learning, adaptive algorithms, running and analyzing performance statistics.

Recommended Skills: Analytics, Algorithms, Data Science, (Golang and/or C++ helpful, but not necessary)

---
**Resources**
- https://smp-spec.io/blog/spec/meshmark-explained

**Comment by anonymousr007** [comment]
Hi @leecalcote ,I am Rishabh, final year Computer Engineering student. I am intermediate level developer in python,c++ and I have good knowledge of data structures and algorithms. I would like to work in this project under LFX Mentorship Fall'21.Can you please provide further information related to this project.

Thanks & Regards
Rishabh

**Comment by DelusionalOptimist** [comment]
Hey @anonymousr007! Thanks for showing interest in this. :slightly_smiling_face: 
Please consider joining https://slack.layer5.io to learn more about this.

**Comment by anonymousr007** [comment]
Which channel is we used for conversation ? I am new in this project and confused 😅

**Comment by anonymousr007** [comment]
Application option show closed of this project : https://mentorship.lfx.linuxfoundation.org/project/278ad0b0-ec8a-474a-863b-a8a01956d99c

**Comment by DelusionalOptimist** [comment]
@anonymousr007 let's start off with a small intro in the newcomers channel :smile:

**Comment by anonymousr007** [comment]
Yeah!!! What's the further process to getting selected in this project under LFX Mentorship Fall'21.

**Comment by abs007** [comment]
Hi, I saw this project in the lfx fall'21 program. I was wondering if this project would be available for the upcoming fall'22 program too.
